### PR TITLE
Utilities:  Added my version of 'dd'

### PR DESCRIPTION
**Pull request by g0mb4**
Recently started to work on Serenity and really missed this tool, so I implemented it:
Screenshot from 2021-03-29 10-13-11

### CODE REVIEW

File: Userland/Utilities/dd.cpp [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+int main(int argc, char** argv)
+{
+    int input_fd = 0;
+    int output_fd = 1;
+    int block_size = 512;
+    int count = -1; // infinite
+
+    // Handling heretic command-line options.
+    for (int a = 1; a < argc; a++) {
```
Comment (chengchenwish) [inline Userland/Utilities/dd.cpp:150]: better to use  Core::ArgsParser to parse args.

Comment (g0mb4) [reply Userland/Utilities/dd.cpp:150]: Ohh, I thought that ArgParser only deals with arguments in the format of "-a value" or "--arg value" and dd uses a different format.

Comment (chengchenwish) [reply Userland/Utilities/dd.cpp:150]: You are right, I didn't aware of this😄.

**Comment by awesomekling** [comment]
Hello and thank you for working on this! Please squash little fixes into the relevant preceding commit instead of adding more commits, we don't need to preserve the fixup history and it just makes it harder to review the PR anyway :)

**Comment by g0mb4** [comment]
Squashed my commits, sorry for the inconvenience :)

**Pull request merged**